.github/workflows/_runner_determinator_script_sync.yaml

```diff
@@ -0,0 +1,42 @@
+name: runner-determinator
```

Comment:
nit: could you please rename the workflow file to remove the underscore prefix? The repo's convention is to use those only when a  workflow is a reusable workflow

Reply:
ah, good catch, thank you

Reply:
fixed